pkg/es/config/config.go

```diff
@@ -127,13 +127,21 @@ func (c *Configuration) NewClient(logger *zap.Logger, metricsFactory metrics.Fac
 
 			sm.Emit(err, time.Since(start.(time.Time)))
 			if err != nil {
-				failed := len(response.Failed())
+				var failed int
+				var respval interface{}
+				if response == nil {
+					failed = 0
+					respval = "nil"
```

Comment:
isn't this logged by default as nil when it's nil?

Reply:
yes , it will be logged as null by default; I will remove unneeded action